src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -1166,35 +1184,6 @@ private static void SubstituteBidiMirrors(FontMetrics fontMetrics, GlyphSubstitu
                         : metric.FontMetrics.VerticalMetrics;
                     float ascender = metricsHeader.Ascender * scaleY;
 
-                    // Adjust ascender for glyphs with a negative tsb. e.g. emoji to prevent cutoff.
-                    if (!CodePoint.IsWhiteSpace(codePoint))
```

Comment:
This was a hack based upon incorrect assumptions. The proper fix is to use a larger line spacing to meet the font design.